supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -311,12 +318,31 @@ object TesAsyncBackendJobExecutionActor {
         getTesStatusFn(Option(state), previousCostData, handle.pendingJob.jobId)
       }
     }
+
+  def getStartAndEndTimes(runStatus: StandardAsyncRunState,
+                          logger: LoggingAdapter,
+                          incrementFn: (InstrumentationPath, Option[String]) => Unit
+  ): Option[StartAndEndTimes] = runStatus.costData match {
+    case Some(TesVmCostData(Some(_ @startTime), Some(_ @endTime), _)) =>
+      try {
+        val parsedStartTime = OffsetDateTime.parse(startTime)
+        val parsedEndTime = OffsetDateTime.parse(endTime)
+        Some(StartAndEndTimes(parsedStartTime, Option(parsedStartTime), parsedEndTime))
+      } catch {
+        case dateTimeParseException: DateTimeParseException =>
+          incrementFn(NonEmptyList.of("parse_tes_timestamp", "failure"), Some("bard"))
+          logger.error(s"Parsing TES task start and end time failed: $dateTimeParseException")
+          None
+      }
```

Comment:
Would a `Try` and `toOption` work here instead of trying to make exceptions work with options?

Reply:
Oh yeah good call!

Reply:
I do want to have some sort of logging/metrics on parsing failures though so I think toOption wont work with that.

Reply:
I did a little refactor to use Try instead